website/docs/cli/import/index.mdx

```diff
@@ -38,13 +38,6 @@ imported object will be mapped.
 While this may seem tedious, it still gives Terraform users an avenue for
 importing existing resources.
 
-## Remote Backends
+## Terraform Cloud
 
-When using Terraform import on the command line with a [remote
-backend](/language/settings/backends/remote), such as Terraform Cloud, the import
-command runs locally, unlike commands such as apply, which run inside your
-Terraform Cloud environment. Because of this, the import command will not have
-access to information from the remote backend, such as workspace variables.
-
-In order to use Terraform import with a remote state backend, you may need to
-set local variables equivalent to the remote workspace variables.
+When you use Terraform on the command line with Terraform Cloud, many commands (e.g., `apply`) run inside your Terraform Cloud environment. However, the `import` command runs locally, so it will not have access to information from Terraform Cloud. To fix this, you may need to set local variables equivalent to any remote workspace variables in Terraform Cloud.
```

Comment:
Well, that doesn't quite land you in a "fixed" state where everything works; it's more just a required hunk of extra work to do the thing. How about: 

```suggestion
When you use Terraform on the command line with Terraform Cloud, many commands (e.g., `apply`) run inside your Terraform Cloud environment. However, the `import` command runs locally, so it will not have access to information from Terraform Cloud. To successfully perform an import, you may need to set local variables equivalent to any remote workspace variables in Terraform Cloud.
```